package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@shopify/functions-test-helpers",
+  "name": "@shopify/shopify-function-test-helpers",
   "version": "1.0.0",
```

Comment:
Do we want to start with a 0.x.y version and then release to 1.0.0 when we're confident it's fairly stable?